packages/block-editor/src/components/collab/block-comment-menu-item.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * WordPress dependencies
+ */
+import { _x } from '@wordpress/i18n';
+import { MenuItem } from '@wordpress/components';
+import { collabComment } from '@wordpress/icons';
+import { useSelect, useDispatch } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { store as blockEditorStore } from '../../store';
+
+export default function BlockCommentMenuItem( { onClose } ) {
+	// eslint-disable-next-line @wordpress/data-no-store-string-literals
+	const { openGeneralSidebar } = useDispatch( 'core/edit-post' );
```

Comment:
This looks concerning beyond using a string literal for the store name. The block-editor package is expected to be WordPress-agnostic, so accessing the edit post store doesn't make sense. This likely hints that this component should live in another package, closer to the post editor. 

This applies to multiple instances in that PR that attempt to use the edit post store in the block editor package.